docs/gno-tooling/cli/gnoland.md

```diff
@@ -4,72 +4,347 @@ id: gno-tooling-gnoland
+| `genesis-remote` | String | A replacement for `$$REMOTES%%` in genesis. (default: `localhost:26657`) |
+| `genesis-txs-file` | String | Initial txs to replay. (default: ~/gno/gno.land/genesis/genesis_txs.jsonl) |
+| `gnoroot-dir` | String | The root directory of the `gno` repository. (default: `~/gno`) |
+| `lazy` | Boolean | Flag indication if lazy init is enabled. Generates the node secrets, configuration, and `genesis.json`. When set to `true`, you may start the chain without any initialization process, which comes in handy when developing. (default: `false`) |
+| `log-format` | String | The log format for the gnoland node. (default: `console`) |
+| `log-level` | String | The log level for the gnoland node. (default: `debug`) |
+| `skip-failing-genesis-txs` | Boolean | Doesn’t panic when replaying invalid genesis txs. When starting a production-level chain, it is recommended to set this value to `true` to monitor and analyze failing transactions. (default: `false`) |
 
-```shell
-gnoland init --help
+### gnoland genesis <subcommand> [flags] [<arg>...]
 
-USAGE
-  init [flags]
+Gno `genesis.json` manipulation suite for managing genesis parameters.
 
-initializes the node directory containing the secrets and configuration files
+#### SUBCOMMANDS
 
-FLAGS
-  -data-dir gnoland-data  the path to the node's data directory
-  -force=false            overwrite existing data, if any
+| Name        | Description                                 |
+| ----------- | ------------------------------------------- |
+| `generate`  | Generates a fresh `genesis.json`.           |
+| `validator` | Validator set management in `genesis.json`. |
+| `verify`    | Verifies a `genesis.json`.                  |
+| `balances`  | Manages `genesis.json` account balances.    |
+| `txs`       | Manages the initial genesis transactions.   |
+
+### gnoland genesis generate [flags]
+
+Generates a node's `genesis.json` based on specified parameters.
+
+#### FLAGS
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `block-max-data-bytes` | Int | The max size of the block data.(default: `2000000`) |
+| `block-max-gas` | Int | The max gas limit for the block. (default: `100000000`) |
+| `block-max-tx-bytes` | Int | The max size of the block transaction. (default: `1000000`) |
+| `block-time-itoa` | Int | The block time itoa (in ms). (default: `100`) |
+| `chain-id` | String | The ID of the chain. (default: `dev`) |
+| `genesis-time` | Int | The genesis creation time. (default: `utc now timestamp`) |
+| `output-path` : | String | The output path for the `genesis.json`. If the genesis-time of the Genesis File is set to a future time, the chain will automatically start at that time if the node is online. (default: `./genesis.json`) |
+
+### gnoland genesis validator <subcommand> [flags]
```

Comment:
We should love out these from "reference" docs to actual tutorials:
- How to generate a genesis.json
- How to generate a config.toml
- How to add a validator to the genesis.json
- ...

You get the idea. Can you propose how we can organize these?

Reply:
Or we can make `technical faq` section and ref thoses, WDYT?

Reply:
ad8a9b0acf291b8ab268915615151e660444b959